src/tokens/optim.ts

```diff
@@ -0,0 +1,19 @@
+import { defaultFetcherOptions, SupplyFetcher } from "../types";
+import { getAmountInAddresses, getBlockFrostInstance } from "../utils";
+
+const OPTIM = "e52964af4fffdb54504859875b1827b60ba679074996156461143dc1";
```

Comment:
Update this to policy + name

`const OPTIM = "e52964af4fffdb54504859875b1827b60ba679074996156461143dc14f5054494d"`